# HG changeset patch
# User kiozen
# Date 1528899896 -7200
# Node ID e9b47af61b0945acb74649bab30dec23ac42f375
# Parent  f52eedfb8f94ea218eb1c55bb8f2fa0c0503238f
[Issue #344 ] Building latest version of QMapShack fails after updating QT from 5.10.1 to 5.11.0

diff --git a/src/qmapshack/helpers/CToolBarSetupDialog.cpp b/src/qmapshack/helpers/CToolBarSetupDialog.cpp
--- a/src/qmapshack/helpers/CToolBarSetupDialog.cpp
+++ b/src/qmapshack/helpers/CToolBarSetupDialog.cpp
@@ -20,6 +20,8 @@
 #include "helpers/CToolBarConfig.h"
 #include "helpers/CToolBarSetupDialog.h"
 
+#include <QAction>
+
 bool CToolBarSetupDialog::CItemFilter::shouldBeMoved(QListWidgetItem *item)
 {
     CDialogItem * dialogItem = dynamic_cast<CDialogItem *>(item);
